requires-testing.txt

```diff
@@ -0,0 +1,9 @@
+pytest<5.0.0
+pytest-sugar
+pytest-mock
+lxml
+selenium
+percy
+requests[security]
+beautifulsoup4
+waitress
```

Comment:
Does `testing` require

dash_flow_example==0.0.5
dash-dangerously-set-inner-html
virtualenv;python_version=="2.7"

if not in the `ci` environment?

Reply:
they should go -ci.txt as only our integration tests need them

Reply:
let me double check the `mock`